README.md

```diff
@@ -72,91 +75,66 @@
+  <a href="https://agenta.ai?utm_source=github&utm_medium=referral&utm_campaign=readme"><b>Website</b></a> &bull;
+  <a href="https://cloud.agenta.ai?utm_source=github&utm_medium=referral&utm_campaign=readme"><b>Agenta Cloud</b></a>
+
 </h3>
 
 ---
 
-# ⭐️ Why Agenta?
-
-Agenta is an end-to-end LLM developer platform. It provides the tools for **prompt engineering and management**, ⚖️ **evaluation**, **human annotation**, and :rocket: **deployment**. All without imposing any restrictions on your choice of framework, library, or model.
-
-Agenta allows developers and product teams to collaborate in building production-grade LLM-powered applications in less time.
-
-### With Agenta, you can:
-
-- [🧪 **Experiment** and **compare** prompts](https://docs.agenta.ai/prompt_management/prompt_engineering) on [any LLM workflow](https://docs.agenta.ai/prompt_management/setting_up/custom_applications) (chain-of-prompts, Retrieval Augmented Generation (RAG), LLM agents...)
-- ✍️ Collect and [**annotate golden test sets**](https://docs.agenta.ai/evaluation/test_sets) for evaluation
-- 📈 [**Evaluate** your application](https://docs.agenta.ai/evaluation/automatic_evaluation) with pre-existing or [**custom evaluators**](https://docs.agenta.ai/evaluation/custom_evaluator)
-- [🔍 **Annotate** and **A/B test**](https://docs.agenta.ai/evaluation/human_evaluation) your applications with **human feedback**
-- [🤝 **Collaborate with product teams**](https://docs.agenta.ai/misc/team_management) for prompt engineering and evaluation
-- [🚀 **Deploy your application**](https://docs.agenta.ai/prompt_management/deployment) in one-click in the UI, through CLI, or through github workflows.
+# What is Agenta?
 
-### Works with any LLM app workflow
+Agenta is a platform for building production-grade LLM applications. It helps **engineering and product teams** create reliable LLM apps faster.
 
-Agenta enables prompt engineering and evaluation on any LLM app architecture:
 
-- Chain of prompts
-- RAG
-- Agents
-
-It works with any framework such as [Langchain](https://www.langchain.com/), [LlamaIndex](https://www.llamaindex.ai/) and any LLM provider (openAI, Cohere, Mistral).
-
-# Quick Start
-
-### [Get started for free](https://cloud.agenta.ai?utm_source=github&utm_medium=readme&utm_campaign=github)
-
-### [Explore the Docs](https://docs.agenta.ai/)
-
-### [Create your first application in one-minute](https://docs.agenta.ai/getting_started/quick-start)
-
-### [Create an application using Langchain](https://docs.agenta.ai/guides/tutorials/first-app-with-langchain)
-
-### [Self-host agenta](https://docs.agenta.ai/self-host/host-locally)
-
-### [Check the Cookbook](https://docs.agenta.ai/guides/cookbooks/evaluations_with_sdk)
+Agenta provides end-to-end tools for the entire LLMOps workflow:  building (**LLM playground**, **evaluation**), deploying (**prompt and configuration management**), and monitoring (**LLM observability and tracing**).
```

Comment:
what do you mean by building (LLM playground?
also evaluation should outside of the ()

Reply:
the three steps are building deploying and monitoring, eval is in the building predeployment step

Reply:
I would also add experimentation